opentimelineio_contrib/adapters/tests/test_aaf_adapter.py

```diff
@@ -39,6 +39,8 @@
 EXAMPLE_PATH5 = os.path.join(SAMPLE_DATA_DIR, "preflattened.aaf")
 EXAMPLE_PATH6 = os.path.join(SAMPLE_DATA_DIR, "nesting_test.aaf")
 EXAMPLE_PATH7 = os.path.join(SAMPLE_DATA_DIR, "nesting_test_preflattened.aaf")
+EXAMPLE_PATH8 = os.path.join(SAMPLE_DATA_DIR, "misc_speed_effects.aaf")
+EXAMPLE_PATH9 = os.path.join(SAMPLE_DATA_DIR, "linear_speed_effects.aaf")
```

Comment:
Could just collapse these example paths into a list comprehension, rather than a list of numerated examples.

Reply:
Hmmm... how about if I name them better than 1,2,3,4...

Reply:
If you put them into a list you can use a comprehension to do the path prefix.  Either way!